src/interfaces/widget/layout.py

```diff
@@ -0,0 +1,55 @@
+"""
+
+    Create a Qt layout wrapper object to arrange child widgets
+    within a widget to ensure that they make good use of the available space.
+
+"""
+
+from PyQt5 import QtWidgets
+
+
+class Layout(QtWidgets.QLayout):
+
+    def __init__(self, widget, **kwargs):
+        super(Layout, self).__init__(widget) #, **kwargs)
+        
+        if not kwargs:
+            pass
+        else:
+            # Lambda func greb input args
+            get_num = lambda x : kwargs[x] if x in kwargs else 0
```

Comment:
This should probably use python's dictionary `get` method. It can be used like so:
```python
# Default to 0 if x is not in kwargs
get_num = lambda x: kwargs.get(x, 0)
```

Note: The same goes for the following line.

Reply:
for `get_param`, can it be:

```python
get_param = lambda x: kwargs.get(x, None)
```

Reply:
Or simplified, 

```python
# return None, if not exist
get_param = lambda x: kwargs.get(x)
```

Reply:
I would suggest adding the `None` just to be consistent with the line above it. Someone unfamiliar with Python may not understand what's going on.